data_collection/advertisements/powerblade_adv_influx.js

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env node
+// original file: https://github.com/lab11/powerblade/tree/master/data_collection/advertisements
+
+var noble = require('noble');
+var config = require('./config');
+var influx;
+
+function getDateTime() {
+
+    var date = new Date();
+
+    var hour = date.getHours();
+    hour = (hour < 10 ? "0" : "") + hour;
+
+    var min  = date.getMinutes();
+    min = (min < 10 ? "0" : "") + min;
+
+    var sec  = date.getSeconds();
+    sec = (sec < 10 ? "0" : "") + sec;
+
+    var year = date.getFullYear();
+
+    var month = date.getMonth() + 1;
+    month = (month < 10 ? "0" : "") + month;
+
+    var day  = date.getDate();
+    day = (day < 10 ? "0" : "") + day;
+
+    return year + ":" + month + ":" + day + ":" + hour + ":" + min + ":" + sec;
+
+}
+
+if (config.connect_to_influx == true) {
+    const Influx = require('influx');
+    influx = new Influx.InfluxDB({
+     host: config.influx.host,
+     port: config.influx.port,
+     database: config.influx.db,
+     username: config.influx.username,
+     password: config.influx.password,
+     schema: [
+       {
+         measurement: config.influx.measurement,
+         fields: {
+           Powerblade_P: Influx.FieldType.FLOAT,
+           Powerblade_E: Influx.FieldType.FLOAT,
+           scaling_factor: Influx.FieldType.FLOAT
+         },
+         tags: [
+           'device',
+           'unit'
+         ]
+       }
+     ]
+    })
+}
+
+var dedup = true;
+if (process.argv.length >= 3 && process.argv[2] == "--all") {
+    dedup = false;
+}
+
+var singleSource = false;
+var pb_address;
+if (process.argv.length >= 4 && process.argv[2] == "--addr") {
+    singleSource = true;
+    pb_address = process.argv[3];
+    console.log("Looking for device " + pb_address);
+}
+
+var UMICH_COMPANY_ID = 0x02E0;
+var POWERBLADE_SERVICE_ID = 0x11;
+var OLD_COMPANY_ID = 0x4908;
+
+// {BLE Address: Most recent sequence number} for each PowerBlade
+var powerblade_sequences = {};
+
+var encoding = 'utf8';
+const fs = require('fs');
+var stream = fs.createWriteStream("powerblade_data.txt", {encoding:encoding,flags:'w'});
```

Comment:
You should add to the README instructions on how to run this script and note that it creates (or overwrites if existing) the file `powerblade_data.txt` with results.